src/overlays/actors/ovl_En_Osn/z_en_osn.h

```diff
@@ -9,9 +9,23 @@ typedef void (*EnOsnActionFunc)(struct EnOsn*, GlobalContext*);
 
 typedef struct EnOsn {
     /* 0x0000 */ Actor actor;
-    /* 0x0144 */ char unk_144[0x90];
+    /* 0x0144 */ ColliderCylinder collider;
+    /* 0X0190 */ SkelAnime skelAnime;
     /* 0x01D4 */ EnOsnActionFunc actionFunc;
-    /* 0x01D8 */ char unk_1D8[0x28];
+    /* 0x01D8 */ Vec3s unk_1D8;
+    /* 0x01DE */ Vec3s unk_1DE;
+    /* 0x01E4 */ char unk_1E4[0x6];
+    /* 0x01EA */ u16  unk_1EA;
+    /* 0x01EC */ u8 unk_1EC;
+    /* 0x01ED */ u8 unk_1ED;
+    /* 0x01EE */ s16 cutscene;
+    /* 0x01F0 */ u8 unk_1F0;
+    /* 0x01F1 */ char unk_1F1[0x3];
+    /* 0x01F4 */ u16 unk_1F4;
+    /* 0x01F6 */ char unk_1F6[0x2];
+    /* 0x01F8 */ s32 unk_1F8;
+    /* 0x01FC */ u8 unk_1FC;
+    /* 0x01FD */ char unk_1FD[0x3];
 } EnOsn; // size = 0x200
```

Comment:
```suggestion
    /* 0x01E4 */ UNK_TYPE1 unk_1E4[0x6];
    /* 0x01EA */ u16  unk_1EA;
    /* 0x01EC */ u8 unk_1EC;
    /* 0x01ED */ u8 unk_1ED;
    /* 0x01EE */ s16 cutscene;
    /* 0x01F0 */ u8 unk_1F0;
    /* 0x01F1 */ UNK_TYPE1 unk_1F1[0x3];
    /* 0x01F4 */ u16 unk_1F4;
    /* 0x01F8 */ s32 unk_1F8;
    /* 0x01FC */ u8 unk_1FC;
} EnOsn; // size = 0x200
```
2 Comments:
Both `/* 0x01F6 */ char unk_1F6[0x2];` and `/* 0x01FD */ char unk_1FD[0x3];` looks like they could be padding. So you might be able to remove them. (but maybe not, it's worth trying though)
Also, if a variable turns out to be unused, you can switch `char` to `UNK_TYPE1`